test/clj/rems/api/test_email.clj

```diff
@@ -36,7 +36,9 @@
                        read-ok-body)]
           (is (= "OK" body))
           (is (= [{:subject "Applications in progress", :to-user "developer"}
-                  {:subject "Applications in progress", :to-user "handler"}]
+                  {:subject "Applications in progress", :to-user "handler"}
+                  ;; TODO: Do not send reminder emails to bots?
```

Comment:
as long as the bots don't have the `:mail` user attribute, they will get no emails

Reply:
Ok, removed this comment.